src/adapter/src/coord/sequencer/inner/peek.rs

```diff
@@ -687,7 +687,7 @@ impl Coordinator {
 
                 let stage = match pipeline() {
                     Ok((global_lir_plan, used_indexes)) => {
-                        if let Some(explain_ctx) = explain_ctx {
+                        if let PeekContext::Explain(explain_ctx) = peek_ctx {
```

Comment:
Make this one a `match` so when you add CopyTo it's a compile error here. All the others I think can stay as if lets because they are targeting explain specially.